python/lsst/ip/isr/isrTask.py

```diff
@@ -934,6 +934,9 @@ def overscanCorrection(self, exposure, amp):
         dataView = maskedImage.Factory(maskedImage, amp.getRawDataBBox())
         overscanImage = maskedImage.Factory(maskedImage, amp.getRawHorizontalOverscanBBox())
 
+        sctrl = afwMath.StatisticsControl()
+        sctrl.setNumSigmaClip(self.config.overscanNumSigmaClip)
```

Comment:
It's not clear to me why passing a `StatisticsControl` object into `overscanCorrection` is better than passing in a rejection threshold: it doesn't add any information (or are you thinking to expand its use?), so the rejection threshold is more fundamental. It's like passing a `Struct` consisting of a single value.
Convince me?

Reply:
I think that the API in `isrFunctions` should support the `StatisticsControl` as the user can set other options (e.g. masks to ignore);  indeed, we could then add them to `IsrTask` without modifying the API